cvat-canvas/src/typescript/masksHandler.ts

```diff
@@ -326,6 +362,7 @@ export class MasksHandlerImpl implements MasksHandler {
             defaultCursor: 'inherit',
         });
         this.canvas.imageSmoothingEnabled = false;
+        this.drawnObjects = this.createDrawnObjectsArray();
```

Comment:
Initialized as a regular array in `releasePaste` 
So, when drawing after paste, it will not work

Reply:
Fixed